debezium-and-camel-on-openshift/managed-kafka.properties

```diff
@@ -0,0 +1,6 @@
+sasl.mechanism=PLAIN
+request.timeout.ms=20000
+bootstrap.servers=${MK_BOOTSTRAP_SERVERS}
```

Comment:
Let's work on unifying the environemtn variables and approaches - for that we can use both the same values as in quarkus dev guide (if that is something for local env) or ServiceBinding etc.